packages/frontend/package.json

```diff
@@ -50,16 +51,20 @@
   },
   "devDependencies": {
     "@babel/core": "^7.18.10",
-    "@storybook/addon-actions": "^7.0.9",
-    "@storybook/addon-essentials": "^7.0.9",
+    "@storybook/addon-actions": "7.0.9",
```

Comment:
is this a desired change? if so, why?

Reply:
yup, we want to keep the 7.0.9 version as the newer ones have some conflicts with our codebase